src/devices/Mcp3xxx/Mcp3xxx.cs

```diff
@@ -25,12 +20,22 @@ public abstract class Mcp3xxx : Mcp3Base
         /// <param name="spiDevice">Device used for SPI communication</param>
         /// <param name="channelCount">Value representing the number of single ended input channels available on the device.</param>
         /// <param name="adcResolutionBits">The number of bits of resolution for the ADC.</param>
-        public Mcp3xxx(SpiDevice spiDevice, byte channelCount, byte adcResolutionBits) : base(spiDevice)
+        public Mcp3xxx(SpiDevice spiDevice, byte channelCount, byte adcResolutionBits)
+            : base(spiDevice)
         {
             ChannelCount = channelCount;
             _adcResolutionBits = adcResolutionBits;
         }
 
+        /// <summary>
+        /// the number of single ended input channel on the ADC
+        /// </summary>
+        protected byte ChannelCount
```

Comment:
Another one of these wrongly named members.

Reply:
Yup, I wouldn't change this for now, and just left it as is with an exception to the rule since that is probably preferable than to taking the breaking change